subsys/bluetooth/services/bas/Kconfig.bas

```diff
@@ -0,0 +1,22 @@
+# Bluetooth GATT Battery service
+
+# Copyright (c) 2018 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_BAS
+	bool "GATT Battery service"
+
+config BT_BAS_BLS_IDENTIFIER_PRESENT
+	bool "Battery Level Identifier Present"
+	help
+	Enable this option if the Battery Level Identifier is present.
+
+config BT_BAS_BLS_BATTERY_LEVEL_PRESENT
+	bool "Battery Level Present"
+	help
+	Enable this option if the Battery Level is present.
+
+config BT_BAS_BLS_ADDITIONAL_STATUS_PRESENT
+	bool "Additional Battery Status Present"
+	help
+	Enable this option if Additional Battery Status information is present.
```

Comment:
Consider adding a `config BT_BAS_BLS` so that we can optionally enable this entire feature (seems optional since it didn't not exist before). We should guard the inclusion of `bas_bls.c` as well as the characteristic with this.